backend/app/api/endpoints/knowledge.py

```
@@ -614,6 +617,77 @@ def migrate_knowledge_base_to_group(
+                _update_kb_summary_after_deletion,
+                kb_id=data.target_kb_id,
+                user_id=current_user.id,
+                user_name=current_user.user_name,
+                trace_context=trace_ctx,
+            )
+        return result
+    except ValueError as e:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=str(e),
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)